Examples/Physics_applications/free_electron_laser/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _examples-free-electron-laser:
+
+Free-electron laser
+===================
+
+This example shows how to simulate the physics of a free-electron laser (FEL) using WarpX.
+In this example, a relativistic electron beam is sent through an undulator (represented by an external,
+oscillating magnetic field). The radiation emitted by the beam grows exponentially
+as the beam travels through the undulator, due to the Free-Electron-Laser instability.
+
+The parameters of the simulation are taken from section 5.1 of :cite:t:`ex-Fallahi2020`.
```

Comment:
Why is there an `ex-` here and in the references below?

Reply:
Hm, I am not quite sure. I followed the same citation format as here:
https://github.com/ECP-WarpX/WarpX/blob/development/Examples/Physics_applications/laser_ion/README.rst

Reply:
Okay, I will check separately what that means and whether it's the right way for citing references with reST.